src/UnROOT.jl

```diff
@@ -44,4 +44,12 @@ include("iteration.jl")
 include("custom.jl")
 include("displays.jl")
 
+@static if VERSION >= v"1.9"
```

Comment:
This should help for any version of julia, no?  Also, top-level `@static` does nothing: top-level expressions in a packages are evaluated at compile-time anyway.

Reply:
this doesn't help for <1.8